In [24]:
import pandas as pd


In [25]:
df = pd.read_csv('traefik.csv',sep=',')

In [26]:
df

data1       data2             ip  status metodo  \
0        2022-01-06 06:01:46  2022-01-06  216.245.221.0     200   HEAD   
1        2022-01-06 06:01:46  2022-01-06  216.245.221.0     200   HEAD   
2        2022-01-06 06:01:46  2022-01-06  216.245.221.0     200   HEAD   
3        2022-01-06 06:01:47  2022-01-06  216.245.221.0     200   HEAD   
4        2022-01-06 06:01:46  2022-01-06  216.245.221.0     200   HEAD   
...                      ...         ...            ...     ...    ...   
4631782  2021-09-27 08:09:38  2021-09-27    54.94.159.0     404    GET   
4631783  2021-09-27 08:09:53  2021-09-27    54.94.159.0     404    GET   
4631784  2021-09-27 08:09:12  2021-09-27    54.94.159.0     404    GET   
4631785  2021-09-27 08:09:14  2021-09-27    54.94.159.0     404    GET   
4631786  2021-09-27 08:09:53  2021-09-27  216.245.221.0     200   HEAD   

                                                   recurso  
0                                                  /main/   
1                                                  /main/   
2                                                  /main/   
3                                                  /main/   
4                                                  /main/   
...                                                    ...  
4631782  /remote/error?errmsg=ABABAB--%3E%3Cscript%3Eal...  
4631783  /WidgetHandler.ashx?MethodName=Sort&ID=1&row=1...  
4631784                                             /%2f/   
4631785                           /sample-apps/hello/%2f/   
4631786                                            /main/   

[4631787 rows x 6 columns]

## Perguntas
* Existe algum padrão de horário ?
* Algum IP específico ? (agrupar)
* Padrões de recursos ? (agrupar por recurso)
* Quantos métodos diferentes ? (para o status 200 e 404). Considere apenas entre / /

## Gráficos
* Acessos Por data (totais, e por método)
* Por método
* Por status, método